Analysis/Tasks/PWGHF/HFCandidateCreatorCascade.cxx

```diff
@@ -44,7 +44,12 @@ struct HFCandidateCreatorCascade {
   Configurable<double> d_minParamChange{"d_minParamChange", 1.e-3, "stop iterations if largest change of any X is smaller than this"};
   Configurable<double> d_minRelChi2Change{"d_minRelChi2Change", 0.9, "stop iterations is chi2/chi2old > this"};
   Configurable<bool> b_doValPlots{"b_doValPlots", true, "do validation plots"};
-
+#ifdef MY_DEBUG
+  Configurable<std::vector<int>> v_labelK0Spos{"v_labelK0Spos", {729, 2866, 4754}, "labels of K0S positive daughters, for debug"};
+  Configurable<std::vector<int>> v_labelK0Sneg{"v_labelK0Sneg", {730, 2867, 4755}, "labels of K0S positive daughters, for debug"};
+  Configurable<std::vector<int>> v_labelProton{"v_labelProton", {729, 2810, 4393}, "labels of K0S positive daughters, for debug"};
```

Comment:
Minor thing: the help strings should be different.

Reply:
Of course.